lib/PAUSE/package.pm

```diff
@@ -293,81 +270,45 @@ sub examine_pkg {
   if ($pp->{version} && $pp->{version} =~ /^\{.*\}$/) { # JSON parser error
       my $err = JSON::jsonToObj($pp->{version});
       if ($err->{openerr}) {
-          $self->index_status($package,
-                              "undef",
-                              $pp->{infile},
-                              PAUSE::mldistwatch::Constants::EOPENFILE,
-
-                              qq{The PAUSE indexer was not able to
-        read the file. It issued the following error: C< $err->{openerr} >},
-                              );
-      } else {
-          $self->index_status($package,
-                              "undef",
-                              $pp->{infile},
-                              PAUSE::mldistwatch::Constants::EPARSEVERSION,
-
-                              qq{The PAUSE indexer was not able to
-        parse the following line in that file: C< $err->{line} >
-
-        Note: the indexer is running in a Safe compartement and cannot
-        provide the full functionality of perl in the VERSION line. It
-        is trying hard, but sometime it fails. As a workaround, please
-        consider writing a META.yml that contains a 'provides'
-        attribute or contact the CPAN admins to investigate (yet
-        another) workaround against "Safe" limitations.)},
-
-                              );
+          # TODO: get $err->{openerr} back in here, I guess?
```

Comment:
Is this todo something to worry about?

Reply:
I think it's okay to leave like this, but … I dug into this whole hunk of code and HOO BOY.  This is a whole 'nother project for a whole 'nother time.